### PR TITLE
Add Vade Cloud connector

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**

### CODE REVIEW

File: VadeCloud/logo.png
Comment (squioc) [inline VadeCloud/logo.png:1]: Can we have the same logo that the one for the format?

File: VadeCloud/vadecloud_modules/client/retry.py
Comment (squioc) [inline VadeCloud/vadecloud_modules/client/retry.py:1]: Does Vade define specific rate-limit headers? Is `Retry` from `requests.adapter` not enough?

File: VadeCloud/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```
Comment (squioc) [inline VadeCloud/CHANGELOG.md:8]: Could you add an entry in the changelog (like [this one](https://github.com/SEKOIA-IO/automation-library/blob/746698e4608fb627f1922f4825cfe5c3d9c9256a/Duo/CHANGELOG.md?plain=1#L23-L27) in Duo)?

**Review by squioc — COMMENTED** [review]
Nice works.

File: VadeCloud/vadecloud_modules/client/retry.py
Comment (lvoloshyn-sekoia) [reply VadeCloud/vadecloud_modules/client/retry.py:1]: No, they don't provide any rate limiting info at all. I'll remove this, thanks

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**